Resources/Prototypes/_Crescent/Roles/Jobs/NCWL/administrator.yml

```diff
@@ -77,6 +77,9 @@
     id: AdministratorPDA
     neck: ClothingNeckAdminCapelet
     ears: ClothingHeadsetNCWL
-    pocket2: WeaponPistolT91
-    shoes: ClothingShoesBootsJack
+    gloves: ClothingHandsGlovesNCWLAdmiralParade
+    belt: ClothingBeltNCWLDoubleHolster
+    pocket1: WeaponRevolverPython
+    pocket2: WeaponRevolverPython
```

Comment:
Unmentioned change.

Reply:
Do your fucking PR descriptions properly , im getting tired of you sneaking in unmentioned changes. I am going to unironically git ban you if this continues.